src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1523,6 +1533,7 @@ public abstract partial class TextWriter : System.MarshalByRefObject, System.IDi
         public virtual System.Threading.Tasks.Task WriteAsync(char value) { throw null; }
         public System.Threading.Tasks.Task WriteAsync(char[] buffer) { throw null; }
         public virtual System.Threading.Tasks.Task WriteAsync(char[] buffer, int index, int count) { throw null; }
+        public virtual System.Threading.Tasks.Task WriteAsync(ReadOnlyMemory<char> source, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
> Using the full type name is consistent with how the ref generator tool generates the code.

There are some places where the full type name is not being used.

Reply:
I'll fix those.